config/sidekiq.yml

```diff
@@ -34,18 +34,6 @@
   digest_run_completion_marker:
     every: '1m'
```

Comment:
While we're hear, I do wonder if we can reduce the frequency of some of these jobs. Perhaps metrics collection can stay at `1m`, but I can't imagine we'd lose much if digests run every 5 minutes instead.

Reply:
That's true. I noticed that previously and thought it a bit weird. That said, I'm going to leave it as-is for now. We've got a stream of work to stabilise the system and we'll be looking at what we can do to make it less surprising as part of that.